src/applications/disability-benefits/all-claims/config/schema.js

```diff
@@ -78,6 +77,10 @@ const documentTypes526 = [
     label:
       'VA Form 21-8940 - Veterans Application for Increased Compensation Based on Un-employability',
   },
+  {
+    value: 'L015',
+    label: 'VA Form 21-4138 - Buddy/Lay Statement',
+  },
```

Comment:
Would you please move this back to the first entry?

Reply:
Done :)